index.css

```diff
@@ -0,0 +1,352 @@
+.wrapper {
+  position: relative;
+  width: 100%;
+  min-height: 38rem;
+
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-grow: 1;
+}
+
+.round-image {
+  position: absolute;
+  top: 2rem;
+  right: -3.4rem;
+  width: 100%;
+}
+.round {
+  position: absolute;
+  top: 0rem;
+  right: 0rem;
+  width: 8rem;
+  height: 8rem;
+  border-radius: 50%;
+  border: none;
+  outline: none;
+  object-fit: cover;
+}
+.card-img img {
+  width: 100%;
+  height: 40rem;
+}
+.cards-example {
+  position: relative;
+
+  width: 35rem;
+}
+
+/* card with exit button */
+
+.card-exit {
+  display: flex;
+  flex-direction: column;
+
+  position: relative;
+  width: 30rem;
+
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+}
+.exit-button {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 1rem;
+  margin-right: 0.5rem;
+  border: none;
+  margin-top: 0.5rem;
+  background-color: white;
+  border-radius: 50%;
+  z-index: 1;
+}
+.cross-image {
+  max-height: 30rem;
+  object-fit: fill;
+}
+.exit-button:hover {
+  background: cadetblue;
+}
+.card-img img {
+  width: 100%;
+  height: 30rem;
+}
+.cart-button {
+  padding: 0.8rem;
+  /* width:10rem;
+          height:3rem; */
+
+  border: none;
+  outline: none;
+  background: black;
+  color: cornsilk;
+}
+.cart {
+  margin: 1rem;
+  padding: 1rem;
+}
+
+.card-body,
+.original-price,
+.discount-price,
+.price {
+  font-size: 2rem;
+}
+
+/* button component */
+
+.button-container-button {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  width: 15rem;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
+
+  color: black;
+  font-family: "roboto";
+}
+.primary-button {
+  background-color: #49a0eb;
+}
+.secondary-button {
+  background-color: #63edff;
+  color: black;
+  font-family: "roboto";
+}
+.danger-button {
+  background-color: red;
+  color: white;
+  font-family: "roboto";
+}
+.toggle-button {
+  position: relative;
+
+  width: 16rem;
+  height: 5rem;
+  border: none;
+
+  background-color: #63edff;
+  color: black;
+  font-family: "roboto";
+}
+.icon-badge-number {
+  position: absolute;
+  top: 0;
+  right: -2.1rem;
+  width: 2.2rem;
+  height: 2.2rem;
+  background: #209bab;
+  color: black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+}
+.toggle-badge {
+  width: 16rem;
+  height: 5rem;
+  position: relative;
+}
+.hide {
+  display: none;
+}
+.floating-action-button {
+  margin-top: 2rem;
+  width: 5rem;
+  height: 5rem;
+  border-radius: 50%;
+  border: none;
+  background-color: #49a0eb;
+  z-index: 1;
+}
+.floating-action-button-small {
+  margin-top: 2rem;
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  border: none;
+  background-color: #49a0eb;
+}
+.floating-action-button:hover,
+.floating-action-button-small:hover {
+  background-color: #63edff;
```

Comment:
Suggestion: You can declare this color in the root if it's used multiple times, for better code structure and readability